packages/create-package/index.js

```diff
@@ -72,9 +72,16 @@ const copyFiles = (vars, sourcePath, destPath) => {
 // Main function
 const createWebcomponentsPackage = async () => {
 	let response;
+	let typescript = false;
 
 	// Get the name
 	let name = process.argv[2];
+	// Get the port
+	let port = process.argv[3];
```

Comment:
let's get rid of the port. it was necessary when it was hardcoded and could collide with openui5. now the dev-server automatically picks the next available port so no need to ask here.

but you should leave the port in the generated output, so whoever wants to change it can find it in the config.

Reply:
Fine by me